src/locale/locales/tr/messages.po

```diff
@@ -3531,7 +3531,7 @@ msgstr "Kullanıcı adı"
 
 #: src/view/screens/AccessibilitySettings.tsx:118
 msgid "Haptics"
-msgstr ""
+msgstr "Dokunuş"
```

Comment:
For the native turkish speakers:
iPhone cihazlarda `Sounds & Haptics`, `Ses ve Dokunuş` olarak çevrildiği için aynı şekilde çevrilmiştir. @usirin'e yönlendirmeleri için teşekkür ederim.

Reply:
Dokunuş daha etkili gibi gözükse de Ses ile birlikte kullanılması gerektiğinde Dokunma daha iyi gibi

Reply:
Şu an sesli geri bildirimden ziyade titreşim olarak geri bildirim var. Eğer bir problem görmüyorsak, aynı şekilde kalsa nasıl olur?

![Screenshot_20241121_222306_Bluesky.jpg](https://github.com/user-attachments/assets/4245807b-9cfb-4ea0-9f62-fb17e283d99c)